app/client/src/features/events/EventLive.tsx

```diff
@@ -80,6 +78,7 @@ export interface EventLiveProps {
 
 export function EventLive({ eventLiveQueryRef, validateInviteQueryRef }: EventLiveProps) {
     const { node } = usePreloadedQuery(EVENT_LIVE_QUERY, eventLiveQueryRef);
+    const data = useFragment(EVENT_SIDEBAR_FRAGMENT, node);
```

Comment:
This data is of unknown type so it is throwing errors when typechecking. If you notice it's pulling a fragment from lower in the hierarchy which is not what you want to do with relay. You should get the fragment ref from the eventLiveQueryRef and remove it from the sidebar fragment if it's not needed there. Basically as a rule of thumb fragment refs should come from a parent and not from siblings. If you notice here node can be null as well which is why the data is unknown.